src/main/java/com/fasterxml/jackson/databind/jsonschema/JsonSerializableSchema.java

```diff
@@ -10,61 +10,64 @@
 /**
  * Annotation that can be used to define JSON Schema definition for
  * the annotated class.
- *<p>
+ * <p>
  * Note that annotation is often not needed: for example, regular
  * Jackson beans that Jackson can introspect can be used without
  * annotations, to produce JSON schema definition.
- * 
+ *
  * @author Ryan Heaton
  * @author Tatu Saloranta
+ * @deprecated Since 2.2, we recommend use of external
+ * <a href="https://github.com/FasterXML/jackson-module-jsonSchema">JSON Schema generator module</a>
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @JacksonAnnotation
+@Deprecated
 public @interface JsonSerializableSchema
 {
     /**
      * Marker value used to indicate that property has "no value";
      * needed because annotations cannot have null as default
      * value.
      */
-    public final static String NO_VALUE = "##irrelevant";
+    String NO_VALUE = "##irrelevant";
 
     /**
      * Property that can be used to indicate id of the type when
      * generating JSON Schema; empty String indicates that no id
      * is defined.
      */
-    public String id() default "";
-    
+    String id() default "";
```

Comment:
Please do not make such mechanical changes; I know it is redundant but these changes obscure actual diff.